endpoints.json

```diff
@@ -67,5 +67,23 @@
         }
       ]
     }
+  },
+  "PATCH /api/articles/:article_id": {
+    "description": "returns updated article with votes increased/decreased by given amount",
+    "queries": ["votes", "article_id"],
+    "exampleResponse": {
+      "article": [
```

Comment:
Dont forget to include an exampleBody that we are expecting the user to provide.